webdriver-spec.html

```diff
@@ -9114,7 +9114,9 @@ <h2 data-dfn-for="NavigatorAutomationInformation">Interface</h2>
  <li><p>If the <a>current top-level browsing context</a> is <a>no longer open</a>,
   return <a>error</a> with <a>error code</a> <a>no such window</a>.
 
- <li>Let <var>root rect</var> be the <a>current top-level browsing context</a>’s
+ <li><p>Wait in an implementation-specific way for layout to complete.
```

Comment:
I think we need to talk about painting and compositor here instead of layout. We might want to base it around requestAnimationFrame which I believe would fire when "layout" is done